collections/_posts/2022-11-26-SparseMultiChannelVAE.md

```diff
@@ -0,0 +1,311 @@
+
+$$D_{KL} (\mathcal{N} (w; \alpha w^2) || p (w) ) ≈ −k_1\sigma(k_2 + k_3 ln \; \alpha) + 0.5 ln(1 + \alpha^{−1}) + k_1 $$
+
+$$k_1 = 0.63576$$, $$k_2 = 1.87320$$, $$k_3 = 1.48695$$, $$\sigma$$(·) Sigmoid function.
+
+The optimization of $$D_{KL}$$ promotes $$\alpha \to \infty $$ ($$\alpha = p/1−p$$), the implicit drop rate $$p$$ tends to 1. The associated weight $$w$$ can be discarded. 
+
+Unless that weight is beneficial for the optimization objective, that is to maximize the data log-likelihood, it will be set to zero.
+**Sparsity arises naturally**.
+
+
+
+## Sparse Multi-Channel VAE
+
+This part will be completed later (After the post on (Sparse) Variational Dropout).
+
+
+
+&nbsp;
+
+# Results
+
+## Medical imaging data
+
+The dataset is composed of
+
+* 504 subjects of the Alzheimer's Disease Neuroimaging Initiative (ADNI) database
+* Clinical channel is complsed by six continuous variables: age, results to mini-mental state examination, adas-cog, cdr, faq tests, scholarship level
+* Three imaging channels: structural MRI, functional FDG-PET, Amyloid-PET. For each modality, 90 image intensities were computed from 90 brain regions mapped in the AAL atlas. This strategy produces 90 features arrays for each image. Lastly, data was centered and standardized across features. 
+
+Their sMCVAE was compared with:
+
+* MCVAE - their model without the sparsity constrain
+* iVAEs - learning of independant VAE per channel
+* VAE - learning a single VAE that takes as input all the channels at once
+
+For each model class, multi-layer architectures were tested, ranging from 1 up to 4 layers for the encoding and decoding structures, with a sigmoid activation applied to all but last layer. 
+
+After training, the latent space for each model was used to classify neurological diseases (MCI and Dementia) by means of ***Linear Discriminant Analysis***. 
+
+During inference, as far the sMCVAE or the MCVAE are concerned, each channel was used to compute a latent variable $$\textbf{z}_i$$ and the average value was calculated to populate the final latent space used for the classification task
+
+For the sparse method, they selected the subspace generated by the most relevant latent dimensions identified by variational dropout $$(p<0.2)$$. Thanks to that, they identified 5 optimal latent dimensions
+
+![](/collections/images/smcvae/medical_results_1.jpg)
+
+&nbsp;
+
+The encoding of the test set in the latent space given by the sMCVAE model is depicted in the figure below where the visualization is limited to the 2D subspace generated by the two most relevant dimensions
+
+![](/collections/images/smcvae/medical_results_2.jpg)
+
+This subspace appears stratified by age and disease status, across roughly orthogonal directions. 
+
+&nbsp;
+
+The last figure below illustrates the capacity of the method in reconstructing missing channels by sampling points from two trajectories in the latent space to predict the imaging data channels
```

Comment:
I would argue that the figure does not directly showcases the reconstruction of missing channels, as the model's generative capabilities. So I would propose to rephrase the sentence to something like:
```suggestion
The last figure below illustrates the capacity of the method to reconstruct plausible imaging channels when manipulating an original sample to move along interpretable trajectories in the latent space.
```